### ISSUE TITLE
Sentence splitter not working properly affecting part of speech tagger

### ISSUE THREAD

**Issue by dkatsiros** [eellak/gsoc2018-spacy#7]
### Problem
I tried to run the sentence splitter submodule ([sentence_splitter.py](https://github.com/eellak/gsoc2018-spacy/blob/dev/spacy/lang/el/res/modules/sentence_splitter.py)) but it didn't work in Greek language for me. I tried loading both _el_core_news_sm_ and _el_core_news_md_  and also tried inserting and encoding text in unicode utf-8. However it does not recognize different sentences but sees them as one. At the same time this affects the part of speech tagger. 
Do you have any idea what might the problem be? 

Thanks in advance.
### Environment
spaCy version:    2.1.4                         
Location:        /home/dimitris/.local/lib/python3.6/site-packages/spacy
Platform:         Linux-4.18.0-17-generic-x86_64-with-Ubuntu-18.04-bionic
Python version:   3.6.7                         
Models:           el, en

**Comment by giannisdaras** [comment]
Hello, thanks for reporting this!
Could you please tell from where did you download the models?
Are you using `spacy-nightly`?

**Comment by dkatsiros** [comment]
No, I downloaded the models from https://spacy.io/models/el . Should I use `spacy-nightly`?

**Comment by giannisdaras** [comment]
Could you try uninstall spacy, install spacy-nightly, download the models through nightly and then check again? 
Sorry for the trouble, I need to check if it is a version problem.

**Comment by dkatsiros** [comment]
I tried but I faced the same problem .
In order to install models through spacy-nightly I used: `python3 -m spacy install el_core_news_md` .
Is that correct? Any other suggestion on something that I may did wrong?

**Comment by PanosAntoniadis** [comment]
I am facing the same problem after trying both.

**Comment by pdimiel** [comment]
The problem remains. 
Is the development team working in it or has it been abandoned?
It is kind of important to know because if it is has been abandoned, we should turn to different solutions.
